src/Cygnus/DrushExport/AbstractExport.php

```diff
@@ -738,10 +752,11 @@ protected function importContentNodes($limit = 100)
                 }
 
                 if (null !== ($type = (isset($this->map['Content'][$node->type])) ? $this->map['Content'][$node->type] : null)) {
+                    $this->convertLegacy($node);
                     $this->convertTaxonomy($node);
                     $this->convertScheduling($node);
                     $this->convertFields($node);
-                    $formatted[] = json_decode(json_encode($node), true);
+                    $formatted[] = json_decode(json_encode($node, 512), true);
```

Comment:
512 is the default value for `depth` -- left over debug? Lets drop it if we're not going to customize the value intentionally.